x/staking/legacy/v043/store_test.go

```diff
@@ -61,7 +61,7 @@ func TestStoreMigration(t *testing.T) {
 		{
 			"ValidatorsByPowerIndexKey",
 			v040staking.GetValidatorsByPowerIndexKey(val),
-			types.GetValidatorsByPowerIndexKey(val),
+			types.GetValidatorsByPowerIndexKey(val, sdk.DefaultPowerReduction),
```

Comment:
We should add an in-place store migration to modify the param store to add the PowerReduction set to sdk.DefaultPowerReduction

Reply:
fixed